src/app/CommandHandler.cpp

```diff
@@ -550,6 +553,18 @@ CHIP_ERROR CommandHandler::FinishStatus()
     return CHIP_NO_ERROR;
 }
 
+CHIP_ERROR CommandHandler::RollbackResponse()
+{
+    VerifyOrReturnError(mState == State::Idle || mState == State::AddedCommand || mState == State::AddingCommand,
```

Comment:
Why would we ever want to support rollback in the AddedCommand state?  I don't think we should....  At that point we have successfully encoded a response, no?

Reply:
Seems so, we don't have to rollback until we want to support multiple commands, but I guess we need to do some large refactor. So removed it.